src/main/java/org/folio/qm/controller/ErrorHandling.java

```diff
@@ -1,29 +1,32 @@
 package org.folio.qm.controller;
 
 import static feign.Util.UTF_8;
-
 import static org.folio.qm.util.ErrorUtils.ErrorType.FOLIO_EXTERNAL_OR_UNDEFINED;
 import static org.folio.qm.util.ErrorUtils.ErrorType.INTERNAL;
 import static org.folio.qm.util.ErrorUtils.ErrorType.UNKNOWN;
 import static org.folio.qm.util.ErrorUtils.buildError;
 
-import javax.servlet.http.HttpServletResponse;
-
 import feign.FeignException;
+import javax.servlet.http.HttpServletResponse;
+import javax.validation.ConstraintViolationException;
 import org.apache.commons.lang.StringUtils;
+import org.folio.qm.exception.QuickMarcException;
+import org.folio.spring.exception.NotFoundException;
+import org.folio.tenant.domain.dto.Error;
 import org.springframework.http.HttpStatus;
+import org.springframework.validation.FieldError;
+import org.springframework.web.bind.MethodArgumentNotValidException;
 import org.springframework.web.bind.MissingServletRequestParameterException;
 import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.bind.annotation.ResponseStatus;
 import org.springframework.web.bind.annotation.RestControllerAdvice;
 import org.springframework.web.method.annotation.MethodArgumentTypeMismatchException;
 
-import org.folio.qm.exception.QuickMarcException;
-import org.folio.spring.exception.NotFoundException;
-import org.folio.tenant.domain.dto.Error;
```

Comment:
Please check your import order in IDE. The default order on folio is:
1. static not org.folio.*
2. blank line
2. static   org.folio.*
3. blank line
4. java.* and javax.*
5. blank line
6. not org.folio.*
7. blank line
8. org.folio.*